static/src/javascripts-legacy/projects/common/modules/experiments/tests/bundle-digital-sub-price-test-1.js

```diff
@@ -4,128 +4,81 @@ define([
-        };
-
-        this.setCopy = function() {
-            if (this.thrasher()) {
-                var subTitleEl = document.querySelector('.membership-ab-thrasher_header .sub_title');
-                if (subTitleEl) {
-                    subTitleEl.innerHTML = '<p>We\'re introducing <strong>new ways</strong> to support' + (detect.isBreakpoint({min: 'desktop'}) ? '<br>' : ' ') + 'the Guardian\'s quality journalism and independent voice. Choose to subscribe or contribute today.</p>';
-                }
-            }
-        };
-
-        this.setLink = function(variant) {
-            if (this.thrasher()) {
-                var linkEl = document.querySelector('.membership-ab-thrasher--wrapper .link-button');
-                if (linkEl && linkEl.getAttribute('href')) {
-                    linkEl.setAttribute('href', config.page.membershipUrl + '/bundles?INTCMP=BUNDLE_PRICE_TEST_1M_' + config.page.edition.toUpperCase() + '_' + variant.toUpperCase());
-                }
-            }
-        };
-
-        this.setCaption = function() {
-            if (this.thrasher()) {
-                var captionEl = document.querySelector('.membership-ab-thrasher--wrapper .link-button div p');
-                if (captionEl) {
-                    captionEl.textContent = 'Subscribe now';
-                }
-            }
-        };
+    var buildTemplate = function(variant) {
+        return template(contributionsEpicSingleButton, {
+            linkUrl1: config.page.membershipUrl + '/bundles?INTCMP=' + 'BUNDLE_PRICE_TEST_1M_' + config.page.edition.toUpperCase() + '_' + variant.id.toUpperCase(),
+            componentName: variant.componentName,
+            title: 'Since you’re here…',
+            p1: '… we’ve got a small favour to ask. More people are reading the Guardian than ever, but far fewer are paying for it. Advertising revenues across the media are falling fast. And ' +
+            '<span class="contributions__highlight">unlike many news organisations, we haven’t put up a paywall – we want to keep our journalism as open as we can</span>' +
+            '. So you can see why we need to ask for your help. The Guardian’s independent, investigative journalism takes a lot of time, money and hard work to produce. But we do it because we believe our perspective matters – because it might well be your perspective, too.',
+            p2: 'If everyone who reads our reporting, who likes it, helps to support it, our future would be much more secure.',
+            p3: '',
+            cta1: 'Support the Guardian'
+        });
+    };
 
-        this.showThrasher = function() {
-            var thrasherContainer = this.thrasherContainer();
-            if (thrasherContainer) {
-                if (!thrasherContainer.classList.contains('visible')) {
-                    thrasherContainer.classList.add('visible');
-                }
-            }
-        };
+    return contributionsUtilities.makeABTest({
+        id: 'BundleDigitalSubPriceTest1M',
```

Comment:
Ok, fair enough. It is just my advice that anything you can do to make the post analysis easier and require less context to analyse is better because when people run queries on old tests or have automated dashboards, this sort of thing confuses people and leads to wrong decisions.